ShowOrder.go

```diff
@@ -18,25 +18,31 @@ type ShowOrderResponse struct {
 //
 // OrderLineItem is also the structure for a line item returned within `CreateOrderResponse.Order.LineItems`, `UpdateOrderResponse.Order.LineItems`, `ShowOrderResponse.Order.LineItems`, and `DeleteOrderResponse.Order.LineItems`․
 type OrderLineItem struct {
-	ID                string  `json:"id,omitempty"`
-	Quantity          int     `json:"quantity,omitempty"`
-	ProductIdentifier string  `json:"product_identifier,omitempty"`
-	Description       string  `json:"description,omitempty"`
-	ProductTaxCode    string  `json:"product_tax_code,omitempty"`
-	UnitPrice         float64 `json:"unit_price,omitempty,string"`
-	Discount          float64 `json:"discount,omitempty,string"`
-	SalesTax          float64 `json:"sales_tax,omitempty,string"`
+	ID                json.Number `json:"id,omitempty"`
```

Comment:
The API accepts a string for line item `id`; however, the response is actually a number.  After bubbling up errors from `json.Unmarshal` this issue became visible. With the type changed from `string` to `json.Number`, `ID` may still be passed in as a string, but will not error when converting the response.